File: app/src/main/java/com/hnidesu/taskmanager/utility/ToastUtil.java
package com.hnidesu.taskmanager.utility;

import android.widget.Toast;

import androidx.annotation.StringRes;

import com.hnidesu.taskmanager.MyApplication;

public class ToastUtil {


    public static void ToastShort(String text){
        Toast.makeText(MyApplication.getInstance().getApplicationContext(),text,Toast.LENGTH_SHORT).show();
    }
    public static void ToastLong(String text){
        Toast.makeText(MyApplication.getInstance().getApplicationContext(),text,Toast.LENGTH_SHORT).show();
    }

    public static void ToastShort(@StringRes int resId){
        Toast.makeText(MyApplication.getInstance().getApplicationContext(),resId,Toast.LENGTH_SHORT).show();
    }
    public static void ToastLong(@StringRes int resId){
        Toast.makeText(MyApplication.getInstance().getApplicationContext(),resId,Toast.LENGTH_LONG).show();
    }
}

File: app/src/main/java/com/hnidesu/taskmanager/util/ToastUtil.kt
package com.hnidesu.taskmanager.util

import android.content.Context
import android.widget.Toast

object ToastUtil {
    fun toastShort(context: Context,text: String?) {
        Toast.makeText(context, text, Toast.LENGTH_SHORT).show()
    }

    fun toastLong(context: Context,text: String?) {
        Toast.makeText(context, text, Toast.LENGTH_SHORT).show()
    }

    fun toastShort(context: Context,resId: Int) {
        Toast.makeText(context, resId, Toast.LENGTH_LONG).show()
    }

    fun toastLong(context: Context,resId: Int) {
        Toast.makeText(context, resId, Toast.LENGTH_LONG).show()
    }

}
